couchdbkit/designer/fs.py

```diff
@@ -391,10 +410,12 @@ def push(path, dbs, atomic=True, force=False, docid=None):
         dbs = [dbs]
 
     doc = document(path, create=False, docid=docid)
-    doc.push(dbs, atomic=atomic, force=force)
+    pushed = doc.push(dbs, atomic=atomic, force=force)
     docspath = os.path.join(path, '_docs')
     if os.path.exists(docspath):
+        pushed = True
         pushdocs(docspath, dbs, atomic=atomic)
+    return pushed
```

Comment:
will this be used anywhere?

Reply:
https://github.com/dimagi/commcare-hq/compare/sk/couch-sync-skip?expand=1